.github/workflows/post-merge.yml

```diff
@@ -1,6 +1,7 @@
 name: Post Merge
 
 on:
+  pull_request:
```

Comment:
This will be removed right?

Reply:
oh I only deleted the `#to be deleted` comment 😆

Reply:
fixed

Reply:
Classic 😄